docs/design.md

```diff
@@ -298,37 +298,87 @@ container foo {
 	container bar {
 		leaf union-leaf {
 			type union {
-				type string;
 				type int8;
+				type enumeration {
+					enum ONE;
+					enum TWO;
+				}
 			}
 		}
 	}
 }
 ```
 
-the `bar` container is mapped to:
+The `bar` container can be mapped to one of the following generated Go code
```

Comment:
"one of the code" sounds a little off grammatically. I'd say "The `bar` container can be translated to Go code according to one of the following strategies."

Reply:
Done.